actors/builtin/market/market_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/filecoin-project/specs-actors/actors/builtin"
 	"github.com/filecoin-project/specs-actors/actors/builtin/market"
 	"github.com/filecoin-project/specs-actors/actors/builtin/miner"
+	"github.com/filecoin-project/specs-actors/actors/builtin/power"
+	"github.com/filecoin-project/specs-actors/actors/builtin/reward"
```

Comment:
Made heavy use out of the default of fil circulating supply being zero which drops the min collateral to zero and minimized breakage.